programs/ssl/ssl_client2.c

```diff
@@ -1065,6 +1081,24 @@ int main( int argc, char *argv[] )
                 default: goto usage;
             }
         }
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+        else if( strcmp( p, "tls13_kex_modes" ) == 0 )
+        {
+            if( strcmp( q, "psk_pure" ) == 0 )
```

Comment:
The `_pure` part here and a few lines below doesn't match the usage string. One of them should be adapted.

Reply:
Fixed.